plugins/experimental/access_control/headers.cc

```diff
@@ -133,7 +133,7 @@ getHeader(TSMBuffer bufp, TSMLoc hdrLoc, const char *header, int headerlen, char
  * @return true - OK, false - failed
  */
 bool
-setHeader(TSMBuffer bufp, TSMLoc hdrLoc, const char *header, int headerlen, const char *value, int valuelen)
```

Comment:
Why not change to `string_view` instead of `char* value, int valuelen`? But otherwise it looks fine.

Reply:
Next version :).

Reply:
@SolidWallOfCode thanks for the review and the suggestion sounds good!

This commit was meant to sync up what we added internally during our first integration and the upstream version 8.x in preparation for migration to 8.x. 

I would like to keep them in sync for now and address your recommendation with the next feature PR (I am going to add a new feature soon, at which point we will have ditched the internal plugin version).